### ISSUE TITLE
Permission Issue: Resource Not Accessible by Personal Access

### ISSUE THREAD

**Issue by DeXin64** [microsoft/jacdac#1384]
## Issue Description:

When attempting to create a pull request, I encountered the following issues:

1. **"CREATE PULL REQUEST" / "Creating pull request..."**  
   During the pull request process, the system displayed the message "Creating pull request...", but the submission ultimately failed.

2. **"Resource not accessible by personal access token"**  
   The error message indicates "Resource not accessible by personal access token," meaning the personal access token (PAT) being used may not have sufficient permissions to access or modify the relevant resources.

I followed the steps outlined in the [Microsoft GitHub Docs](https://microsoft.github.io/jacdac-docs/tools/device-registration/). Could you please help me identify if there was any issue in the steps that caused this submission failure? I would appreciate any guidance or assistance.

![Image](https://github.com/user-attachments/assets/c0badb1e-5900-46ed-8ee5-53e4b274cb4e)
![Image](https://github.com/user-attachments/assets/3498045a-22b1-4d63-ac90-d37ff7392a70)
![Image](https://github.com/user-attachments/assets/dcea970f-4a99-4ae6-9124-299f5e04c5a1)
![Image](https://github.com/user-attachments/assets/838429c2-4926-4280-a3b1-394b6326ff26)

**Comment by DeXin64** [comment]
I'm encountering an issue and would greatly appreciate your guidance on how to address it. Could you assist me, please?

**Comment by pelikhan** [comment]
Were you able to create it? I see https://github.com/microsoft/jacdac/pull/1386

**Comment by DeXin64** [comment]
Did I correctly understand that you referred to previously submitted pull requests (PRs) by other users as examples (such as shown in this link), made similar modifications to the relevant files, and attempted to submit them? However, you encountered issues when trying to submit via the Jacdac website, which have been described in this issue. Could you please confirm this point and let us know if further assistance is needed to resolve the problems encountered during the submission process?




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "microsoft/jacdac"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2024年12月5日(星期四) 上午9:21
***@***.***&gt;;
抄送:&nbsp;"Le ***@***.******@***.***&gt;;
主题:&nbsp;Re: [microsoft/jacdac] Permission Issue: Resource Not Accessible by Personal Access (Issue #1384)





 
Were you able to create it? I see #1386

**Comment by DeXin64** [comment]
Did I correctly understand that you referred to previously submitted pull requests (PRs) by other users as examples (such as shown in [this link](https://github.com/microsoft/jacdac/pull/1347/files)), made similar modifications to the relevant files, and attempted to submit them? However, you encountered issues when trying to submit via the Jacdac website, which have been described in [this issue](https://github.com/microsoft/jacdac/issues/1384). Could you please confirm this point and let us know if further assistance is needed to resolve the problems encountered during the submission process?

**Comment by pelikhan** [comment]
For now, you can submit your module by manually creating PRs until we figure out what is going on.

**Comment by DeXin64** [comment]
Sure, no problem. Could you please let me know when the web-based PR submission functionality is expected to be restored?

**Comment by DeXin64** [comment]
Sure, no problem. Could you please let me know when the web-based PR submission functionality is expected to be restored?

**Comment by pelikhan** [comment]
Could you try using a classic token? (not fined grained)

**Comment by DeXin64** [comment]
![Image](https://github.com/user-attachments/assets/4e913119-1c93-42dd-9cfb-c18bcf41a932)

**Comment by pelikhan** [comment]
Is your DeXin64/jacdac repo public? I think it might have to be public.

**Comment by DeXin64** [comment]
![Image](https://github.com/user-attachments/assets/b04a4520-720f-448f-bb72-b639c59f548f)
Yes, this repository is public.

**Comment by DeXin64** [comment]
Could you please advise if there are any other troubleshooting methods?

**Comment by pelikhan** [comment]
Manual PR submission is probably the best for now.

**Comment by DeXin64** [comment]
Okay, no problem.

**Comment by pelikhan** [comment]
Are you registering more modules?
________________________________

**Comment by DeXin64** [comment]
Yes, I initially uploaded the first module to streamline and standardize the listing process. Subsequently, I plan to upload the remaining modules together. Thank you for your assistance

**Comment by pelikhan** [comment]
Let's do this by creating pull requests manually - so you're not blocked

**Comment by DeXin64** [comment]
Okay, understood. I will proceed with manually creating the pull request immediately. If there are any issues or if further information is needed, please let me know at any time.

Thank you very much for your guidance!

**Comment by pelikhan** [comment]
I will review them and add comments as needed
